steps/xcode-build-version/0.0.3/step.yml

```diff
@@ -0,0 +1,58 @@
+title: Get Xcode project version
+summary: |
+  Gets version and build number of the project
+description: |
+  Gets project's version and build number
+website: https://github.com/nodes-ios/bitrise-step-xcode-build-version
+source_code_url: https://github.com/nodes-ios/bitrise-step-xcode-build-version
+support_url: https://github.com/nodes-ios/bitrise-step-xcode-build-version/issues
+published_at: 2020-05-11T10:07:18.599768+01:00
+source:
+  git: https://github.com/nodes-ios/bitrise-step-xcode-build-version.git
+  commit: b3adc05db9188ef24126307f5b8c5aab1654d681
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+- macos
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- bitrise_tag_xcodeproj_path: ""
```

Comment:
I would recommend not including the bitrise prefix in the input name as is not related to the Bitrise platform itself. So it could be `project_path`. Similarly for info_plist_path.